examples/platform/k32w/app/support/Memconfig.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#ifndef SRC_PLATFORM_K32W_MEMCONFIG_H_
+#define SRC_PLATFORM_K32W_MEMCONFIG_H_
+
+extern "C" {
+size_t xPortMallocUsableSize(void * pv);
+void * __wrap_malloc(size_t size);
```

Comment:
These declarations are unused and should never need to be, so I'd suggest just omitting them.

Reply:
Fixed